test/test_genericprocess.py

```diff
@@ -0,0 +1,70 @@
+# pylint: disable=missing-docstring
+# -*- coding: utf-8 -*-
+
+"""
+Copyright 2019 ARM Limited
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import subprocess
+import unittest
+import platform
+import os
+
+from icetea_lib.IceteaManager import ExitCodes
+
+
+class GenericProcessTestcase(unittest.TestCase):
+    @unittest.skipIf(
+        platform.system() == "Windows",
+        "Launching process in Windows ends in icetea warning"
+        "\"This OS is not supporting select.poll() or select.kqueue()\"")
+    def test_quick_process(self):
+        # Windows doesn't have /bin/echo, use "cmd -c echo <test string>"
+        # from different configration
+        tc_cfg = os.path.join(
+            os.path.dirname(__file__), "tests",
+            "test_genericprocess_tc_cfg_unix.json")
+        if platform.system() == "Windows":
```

Comment:
this probably is not needed since test is skipped anyway in windows

Reply:
But if the Windows is fixed, then it will be ready.